src/pages/name-it-and-they-will-come/index.pt-br.md

```diff
@@ -0,0 +1,98 @@
+Finalmente, é o dia do lançamento. Você publica o projeto no GitHub. Você twitta sobre e envia a landing page para agregadores populares de notícias sobre código aberto.
+
+```bash
+git push origin master
+npm publish
+```
+
+**Você está ansioso para ouvir o mundo sobre a sua ideia.**
+
+Talvez eles amem. Talvez eles odeiem.
+
+Tudo que você sabe é que ela merece ser ouvida.
+
+---
+
+Parabéns!
+
+Seu projeto atingiu a primeira página de um agregador de notícias popular. Alguém visível na comunidade twittou sobre também. O que estão dizendo?
+
+Seu coração aperta.
+
+Não é que as pessoas *não gostaram* do projeto. Você sabe que houveram tradeoffs e esperava que as pessoas comentassem sobre eles. Mas não foi isso que aconteceu.
+
+**Em vez disso, os comentários são em grande parte *irrelevantes* para a sua ideia.**
+
+As principais discussões e comentários são sobre o estilo do código em um exemplo no README. Que se transformaram em uma discussão sobre indentação com mais de cem respostas e um breve histórico de como diferentes linguagens de programação abordavam a formatação. Existem menções obrigatórias de gofmt e Python. Vocês já usaram Prettier?
+
+Alguém menciona que projetos de código aberto não devem ter landing pages bonitas porque é marketing enganoso. E se um desenvolvedor júnior se apegar a isso sem entender completamente os fundamentos?
+
+Em uma resposta, alguém argumenta que o design da landing page é entediante. Além disso, está quebrado no Firefox. Claramente, isto significa que o autor do projeto não se preocupa com a Internet livre. A Internet como conhecemos está morrendo? É hora dos jogos sobre teorias...
+
+O próximo comentário é uma observação genérica sobre a natureza das abstrações, e como elas podem levar a muito “boilerplate” (ou, alternativamente, “mágica”). As principais respostas explicam que não se deve confundir “simples” com “fácil”. Na verdade, Rich Hickey fez uma palestra muito boa sobre isso. Vocês assistiram?
+
+Finalmente, por que precisamos de bibliotecas? Algumas linguagens funcionam bem com as bibliotecas internas padrão. O npm é um erro? O incidente com o leftpad pode acontecer novamente. Nós devemos realizar o build do npm no navegador? O que acontece com os padrões?
+
+**Confuso, você fecha a aba.**
+
+---
+
+O que aconteceu?
+
+---
+
+Pode ser que sua ideia simplesmente não é tão interessante quanto você pensou. Acontece. Também pode ser que você tenha explicado mal para um visitante casual.
+
+No entanto, pode haver outro motivo pelo qual você não recebeu feedbacks relevantes.
+
+**Nós tendemos a discutir coisas que são fáceis de falar sobre.**
+
+Experiências compartilhadas universais são fáceis de falar sobre. Isto inclui tópicos como formatação de código, verbosidade vs mágica, configuração vs convenção, diferenças nas culturas da comunidade, escândalos, intrevistas técnicas, fofocas da indústria, tendências macro e opiniões de design. Nós temos um vocabulário compartilhado para todas essas coisas.
```

Comment:
Hi @celso-henrique,
Found a typo in this sentence:
`intrevistas` -> `entrevistas`

Reply:
Ops, I fixed that, thanks